cmd/gcs/main.go

```diff
@@ -1,8 +1,10 @@
+//go:build linux
```

Comment:
why do we have this?

Reply:
It looks like gofmt is adding them manually, since go build decided to [switch](https://pkg.go.dev/cmd/go#hdr-Build_constraints) to this new syntax:
> Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax.

Reply:
oh weird

Reply:
Yep, added automatically on 1.17 it seems

Reply:
Maybe we should update everything to use 1.17 soon haha I'm not sure if we should keep this in until that's done. Thoughts?

Reply:
would we have to disable `go fmt` then, since its automatically added by it?

Reply:
I don't believe there's any harm in having both types of build tags, 1.16 and prior won't know what they are, and 1.17 and newer will prefer the new version. Ideally we should run `go fmt` on our whole codebase to have it add the new syntax of build tags to everything. I can do this shortly

Reply:
https://github.com/microsoft/hcsshim/pull/1222